plugins/splunk-on-call/src/components/SplunkOnCallCard.tsx

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React, { useState, useCallback } from 'react';
+import {
+  useApi,
+  Progress,
+  HeaderIconLinkRow,
+  MissingAnnotationEmptyState,
+  configApiRef,
+  EmptyState,
+} from '@backstage/core';
+import { Entity } from '@backstage/catalog-model';
+import {
+  Button,
+  makeStyles,
+  Card,
+  CardHeader,
+  Divider,
+  CardContent,
+  Typography,
+} from '@material-ui/core';
+import { Incidents } from './Incident';
+import { EscalationPolicy } from './Escalation';
+import { useAsync } from 'react-use';
+import { Alert } from '@material-ui/lab';
+import { splunkOnCallApiRef, UnauthorizedError } from '../api';
+import AlarmAddIcon from '@material-ui/icons/AlarmAdd';
+import { TriggerDialog } from './TriggerDialog';
+import { MissingApiKeyOrApiIdError } from './Errors/MissingApiKeyOrApiIdError';
+import { User } from './types';
+
+const useStyles = makeStyles({
+  triggerAlarm: {
+    paddingTop: 0,
+    paddingBottom: 0,
+    fontSize: '0.7rem',
+    textTransform: 'uppercase',
+    fontWeight: 600,
+    letterSpacing: 1.2,
+    lineHeight: 1.5,
+    '&:hover, &:focus, &.focus': {
+      backgroundColor: 'transparent',
+      textDecoration: 'none',
+    },
+  },
+});
+
+export const SPLUNK_ON_CALL_TEAM = 'splunk-on-call.com/team';
```

Comment:
Wondering whether this is a good annotation tbh, `splunk-on-call.com` isn't an actual domain. Thoughts on something like `splunk.com/on-call-team`?